src/js/shield_defs.js

```diff
@@ -500,7 +500,7 @@ export function loadShields() {
       textColor: Color.shields.brown,
       colorLighten: Color.shields.brown,
     },
-    ["HIST"],
+    ["XYZA"], //Change to be removed before merging
```

Comment:
I will remove this change before merging.  Leaving it in for now so we can see an example of a map change.